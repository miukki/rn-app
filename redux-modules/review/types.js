export const RENDER_EMPTY = '@@review/RENDER_EMPTY'
export const RENDER_START = '@@review/RENDER_START'
export const RENDER_RELOAD = '@@review/RENDER_RELOAD'
export const UPDATE_REMAIN_AMOUNT = '@@review/UPDATE_REMAIN_AMOUNT'
