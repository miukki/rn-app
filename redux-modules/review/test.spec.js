import reducer from './index'
import * as types from './types'

describe('FlipCards reducer', () => {
  it('should return the initial state', () => {
    expect(reducer(undefined, {})).toMatchSnapshot()
  })

  it('should handle action.type RENDER_EMPTY', () => {
    expect(
      reducer(reducer(undefined, {}), {
        type: types.RENDER_EMPTY
      }).toJS()
    ).toMatchSnapshot()
  })
})
