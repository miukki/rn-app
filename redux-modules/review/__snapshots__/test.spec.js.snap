// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FlipCards reducer should handle action.type RENDER_EMPTY 1`] = `
Object {
  "items": Array [],
  "planAmount": 0,
  "remain": 0,
  "warning": "No Learned Words",
}
`;

exports[`FlipCards reducer should return the initial state 1`] = `
Immutable.Map {
  items: Immutable.List [
  ],
  warning: "No Learned Words",
  planAmount: 0,
  remain: 0,
}
`;
